src/org/thoughtcrime/securesms/ConversationFragment.java

```diff
@@ -867,6 +869,12 @@ public void onShowFullClicked(DcMsg messageRecord) {
       public void onDownloadClicked(DcMsg messageRecord) {
         dcContext.downloadFullMsg(messageRecord.getId());
       }
+
+      @Override
+      public void onReactionClicked(DcMsg messageRecord) {
+        ReactionsDetailsFragment dialog = new ReactionsDetailsFragment(messageRecord.getId());
+        dialog.show(((FragmentActivity) getActivity()).getSupportFragmentManager(), null);
```

Comment:
the cast to `FragmentActivity` activity is not needed say android studio, and indeed it works without.

at line 50, `import androidx.fragment.app.FragmentActivity;` can be removed as well then